### ISSUE TITLE
Локализация

### ISSUE THREAD

**Issue by RomkaZVO** [saber-nyan/ddlc-android#8]
Есть русик http://www.zoneofgames.ru/games/doki_doki_literature_club/files/5741.html сделай с ним пожалуйста, хотел сам но не вышло.

**Comment by saber-nyan** [comment]
О, уже вышел. Там что ли несколько команд одновременно делали?
Займусь.

**Comment by saber-nyan** [comment]
**AWW SHIT MAN**
![logo_ru](https://user-images.githubusercontent.com/25345196/34271911-516031d0-e69f-11e7-935d-08b1d6e425f3.png)

А он точно качественный? (На всякий случай, обновленная [ссылка на перевод](https://yadi.sk/d/bXDLi8ww3QmUBi))

%%Олсо, как же криво они сделали установку. И инсталлер какой-то мутный, не распакуешь. Благо, в последнем патче они сами распаковали.%%

Не, ну я уже даже и хз.
https://github.com/saber-nyan/ddlc-android/blob/c4c6288541b95b5161967dbbf23a4592a235b284/game/definitions.rpy#L1318-L1321

**Comment by saber-nyan** [comment]
@RomkaZVO Точно не хочешь подождать [этих](https://vk.com/doki_doki_literature_club) парней?<br/>
![meh](https://user-images.githubusercontent.com/25345196/34275317-c6115f64-e6ad-11e7-8116-362334612b39.png)

**Comment by RomkaZVO** [comment]
Я бы подождал, да не терпится уже. Кстати, по поводу распаковки, я спокойно инсталлер разобрал.

**Comment by RomkaZVO** [comment]
А вообще, конечно, дело твоё:)

**Comment by saber-nyan** [comment]
Просто он действительно плохой: я посидел, посравнивал с оригиналом. Да и те ребята, вроде, скоро сделают. Короче, я лучше подожду, ты уж извини.

Как инсталлер разбирал, кстати?

**Comment by RomkaZVO** [comment]
![default](https://user-images.githubusercontent.com/15728269/34292191-ca1c97c6-e73a-11e7-8934-12753f238285.PNG)
Вот этим разобрал. А так да, конечно, подожду.

**Comment by saber-nyan** [comment]
Ага, спасибо.

**Comment by saber-nyan** [comment]
Они там [почти живые](https://vk.com/wall-155243863_1583), вычитывают. Это радует, *тук-тук* меня просто убил.

**Comment by saber-nyan** [comment]
Оо, [выпустили](https://vk.com/wall-155243863_1929).
Скоро начну делать!

**Comment by Otorhin** [comment]
Кхм. Я выпустил качественную версию с русиком ^-^

**Comment by Otorhin** [comment]
Даже файлы поддерживает в Android/Data/{Name Package}/Files
Именно все как на ПК =]

**Comment by saber-nyan** [comment]
Ссылочку?

**Comment by Otorhin** [comment]
https://vk.com/wall-141381337_33 , у меня в профиле гита все скрипты

**Comment by RomkaZVO** [comment]
А как же господа без рута удалят монику?

**Comment by saber-nyan** [comment]
Предлагаю совать Монику в `/sdcard/Android/package.name/files`

**Comment by RomkaZVO** [comment]
Ящитаю, что все надо в android/data/пакет нейм пихать

**Comment by Otorhin** [comment]
Я же так и написал -_-

**Comment by RomkaZVO** [comment]
![screenshot_20180105-213927](https://user-images.githubusercontent.com/15728269/34611592-0c356526-f261-11e7-8c73-ad7dc47648de.png)
Забавно, но мою тсусю снес уже, пишу тут

**Comment by Otorhin** [comment]
Так. Я не умею пользоваться гитхабом нормально =[. Ну да ладно. Оно у тебя в data/data? Проверь другие RenPy игры. Как они создают?

**Comment by RomkaZVO** [comment]
У остальных проблем не было. Хотя у меня тогда и другое устройство было. Да, в data/data сидмт

**Comment by Otorhin** [comment]
Хм... Странно. Я сейчас привлек левых людей. У них все в android/data.

**Comment by saber-nyan** [comment]
Да, забыл написать: жду [SUN-TEAM](https://vk.com/sunteam2015). Что-то грустно все с переводами.
Чуть подробнее [здесь](https://github.com/saber-nyan/ddlc-android/wiki/FAQ-RU).

**Comment by saber-nyan** [comment]
[Вышел!](https://vk.com/wall-101862768_5388)
Проверю качество и начну... когда-нибудь.
